pages/1.13/monitoring/logging/logging-api-examples/index.md

```diff
@@ -59,7 +59,7 @@ curl -k -H "Authorization: token=${DCOS_AUTH_TOKEN}" "${DCOS_URL}/system/v1/agen
 
 # Container Logs
 
-**Important:** The following example requires journald task logging, which by default is [disabled](/1.13/monitoring/logging/logging-api/#compatibility).
+**Important:** The following example requires journald task logging, which by default is [disabled](/1.13/monitoring/logging/logging-reference/#compatibility).
```

Comment:
Use the format for `Important message` from https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes